helmfile.d/stacks/rbac.yaml

```diff
@@ -51,7 +51,21 @@ templates:
     labels:
       app: dev-rbac
     values:
-      - values/userCRDs/common.yaml.gotmpl
       - values/userCRDs/bitnami/sealedsecrets.yaml.gotmpl
-      - values/userCRDs/mongodbcommunity/mongodb.yaml.gotmpl
+      - values/userCRDs/common.yaml.gotmpl
       - values/userCRDs/flux/fluxv2.yaml.gotmpl
+      - values/userCRDs/mongodbcommunity/mongodb.yaml.gotmpl
+      - values/userCRDs/strimzi/kafka.yaml.gotmpl
+
+  extra-dev-rbac:
+    inherit: [ template: rbac ]
+    condition: ck8sWorkloadCluster.enabled
+    chart: charts/user-crds
+    version: 1.0.0
+    name: extra-dev-rbac
```

Comment:
Nit: I would prefer `dev-rbac-extra` as name so they would be naturally sorted together.